framework/src/types.ts

```diff
@@ -153,3 +160,21 @@ export interface TransactionJSON {
 	readonly id: string;
 	readonly asset: object;
 }
+
+// TODO: replace definition from lisk-chain after #5609 "Update lisk-chain to support the on-chain architecture"
+export interface Validator {
+	address: Buffer;
+	canVote: boolean;
+}
+
+// TODO: replace definition from lisk-chain after #5609 "Update lisk-chain to support the on-chain architecture"
+export interface AccountSchema {
+	type: string;
+	properties: Record<string, unknown>;
```

Comment:
We should use use `Schema` type from `lisk-codec` here and not keep it open to any attributes.

Reply:
properties is also open for the lisk-codec `Schema` but somehow what is important is there is required property `default` here